.autopep8.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python3
+# TODO: Explain usage here?  Why do we need both pylint and pep8?
```

Comment:
I assume that this all happens through CI but is it also possible to run these directly?

Reply:
Yes, they can be executed normally. VSCode is supposed to call them on save as well but it's not entirely working, see #32